app.js

```diff
@@ -38,24 +33,39 @@ navigator.geolocation.getCurrentPosition(getLocation, handleError);
 // Response format is Json by default.
 // https://openweathermap.org/current --free
 
-const KEY = '3137461397c70af0e13c77fda97afa11';
+const KEY = "3137461397c70af0e13c77fda97afa11";
 
 function getTemperature(latitude, longitude) {
-    const api = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${KEY}&units=metric`;
-    //the fetch() method instructs the web browsers to send a request to a URL.
-    //less than60 calls/min
-    fetch(api)
-        .then ((response) => response.json())
-            // data --> Promise object represents the eventual completion (or failure)
-            // of an asynchronous operation and its resulting value.
-            // console.log(response);
-        .then((data) => {
-            console.log('the data is:', data);
-            document.getElementById('temperature').innerHTML = `The 
-            temperature in your location is ${data.main.temp.toFixed(2)}C°`;
-        })
-        .catch((error) => {
-            // handles errors
-            console.warn(`error: ${error.message}, weather API`);
-        });
+  const api = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${KEY}&units=metric`;
+  //the fetch() method instructs the web browsers to send a request to a URL.
+  //less than60 calls/min
+  fetch(api)
+    .then((response) => response.json())
+    // data --> Promise object represents the eventual completion (or failure)
+    // of an asynchronous operation and its resulting value.
+    // console.log(response);
+    .then((data) => {
+      console.log("the data is:", data);
+      document.getElementById(
+        "temperature"
+      ).innerHTML = `${data.main.temp.toFixed(2)}°C`;
+    })
+    .catch((error) => {
+      // handles errors
+      console.warn(`error: ${error.message}, weather API`);
+    });
 }
+
+//data.main.temp
```

Comment:
I think it's not a good idea to store these comment in js file. You can create a section in your README file with api description and describe there which type of data and in which shape api is returning. And then it's more visible and you can even leave it there and not delete after you finished.

Reply:
This changes will go into the next pull request :)